### DISCUSSION TITLE
Foaling Date Calculator

### DISCUSSION THREAD

**Discussion by zurtri** [django-commons/django-unicorn#218] [Show and tell]
Hi all - my first effort at unicorn was a simple one.

The coding was fairly straight forward although it kept converting the dates to strings. So I had to include a couple of helper methods to convert them to an object type that the Django template wanted.

All in all it seems to have turned out alright:

https://horserecords.info/foaling_date_calculator/

**Comment by adamghill** [comment]
I'm getting an `Unicorn is missing. Do you need {% load unicorn %} or {% unicorn_scripts %}?` error message in the console. :( Are you sure the unicorn javascript is loading correctly?

**Reply by zurtri** [reply]
Ok try now!

Turns out I had to disable the Rocket Loader in Cloudflare.

**Reply by adamghill** [reply]
It's working now and it looks great!
